### PR TITLE
PP-11594 Add ‘sandbox’ to ‘isDigitalWalletSupported’ test.

### PR DESCRIPTION
**Pull request by JFSGDS**
## WHAT
Allow enabling Apple Pay and Google Pay for Sandbox gateway accounts

[https://payments-platform.atlassian.net/browse/PP-11594](https://payments-platform.atlassian.net/browse/PP-11594)

## HOW 
- Extend existing ‘isDigitalWalletsSuported’ test for ‘Stripe’ and ‘Worldpay’ accounts to include ‘sandbox’ accounts
- Add feature flag ‘ALLOW_ENABLING_DIGITAL_WALLETS_FOR_SANDBOX_ACCOUNT’ as part of test condition
- Add unit tests for all sandbox feature flag states

### CODE REVIEW

**Comment by nlsteers** [comment]
can you squash some of these commits and rebase the branch, there's a bunch of merge commits in here that don't need to be

File: app/utils/display-converter.js [hunk 1]
```diff
@@ -122,6 +122,7 @@ module.exports = function (req, data, template) {
   convertedData.isTestGateway = _.get(convertedData, 'currentGatewayAccount.type') === 'test'
   convertedData.isSandbox = paymentProvider === 'sandbox'
   convertedData.isDigitalWalletSupported = paymentProvider === 'worldpay' ||
+    (paymentProvider === 'sandbox' && process.env.ALLOW_ENABLING_DIGITAL_WALLETS_FOR_SANDBOX_ACCOUNT === 'true') ||
     (paymentProvider === 'stripe' && process.env.ALLOW_ENABLING_DIGITAL_WALLETS_FOR_STRIPE_ACCOUNT === 'true') ||
     (paymentProvider === 'stripe' && convertedData.isTestGateway === true)
```
Comment (nlsteers) [inline app/utils/display-converter.js:127]: wallet payments for stripe are now live so we can probably remove the env var that toggles this feature

```suggestion
  convertedData.isDigitalWalletSupported = paymentProvider === 'worldpay' ||
    (paymentProvider === 'sandbox' && process.env.ALLOW_ENABLING_DIGITAL_WALLETS_FOR_SANDBOX_ACCOUNT === 'true') ||
    paymentProvider === 'stripe'
```

Comment (nlsteers) [reply app/utils/display-converter.js:127]: might need to update some tests if taking this suggestion

**Comment by JFSGDS** [comment]
This pull request has been superseded by pull request [#4153 ](https://github.com/alphagov/pay-selfservice/pull/4153) and can now be closed.

**Pull request closed without merging**